tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs

```diff
@@ -59,12 +93,26 @@ public PhpDynamicInstallationTest(ITestOutputHelper output) : base(output)
             {
                 var data = new TheoryData<string, string, string>();
                 var imageHelper = new ImageTestHelper();
+
+                // Test default PHP composer version
                 data.Add(PhpVersions.Php73Version, imageHelper.GetCliImage(),PhpVersions.ComposerVersion);
                 data.Add(PhpVersions.Php74Version, imageHelper.GetCliImage(), PhpVersions.ComposerVersion);
 
-                // test latest php-composer version
+                // Test PHP composer version 2.3.x
                 data.Add(PhpVersions.Php73Version, imageHelper.GetCliImage(), PhpVersions.Composer23Version);
                 data.Add(PhpVersions.Php74Version, imageHelper.GetCliImage(), PhpVersions.Composer23Version);
+
```

Comment:
I'm wondering if we should stop adding tests for stretch. Or maybe even start removing stretch tests when we come across them? Since Stretch has been EOL [for a while](https://endoflife.date/debian) now. If/when the stretch tests were to fail, I wonder if it'll be of value for us to mitigate. We'd also save some pipeline test time by opting out stretch.

Reply:
Spoke offline about this -- I don't think there's any action required in this PR